### PR TITLE
Update v1.7.0

### PR DESCRIPTION
**Pull request by timvisee**
Regenerated the Go client with the latest changes from `dev` for 1.7.0.

I've also improved our synchronization script a bit to edit files in-place with `sed`, rather than piping things around with `grep`.

### CODE REVIEW

File: tools/sync_proto.sh [hunk 1]
```diff
@@ -19,21 +19,19 @@ CLIENT_DIR="proto"
...
- | grep -v 'points_internal_service.proto' \
- | grep -v 'qdrant_internal_service.proto' \
- | grep -v 'raft_service.proto' \
- | grep -v 'health_check.proto' \
-  > $CLIENT_DIR/qdrant_tmp.proto
-
-mv $CLIENT_DIR/qdrant_tmp.proto $CLIENT_DIR/qdrant.proto
-
-
+sed -i '
```
Comment (agourlay) [inline tools/sync_proto.sh:28]: I think we should ask both snapshot services to this list

**Comment by agourlay** [comment]
needs a regen

**Comment by agourlay** [comment]
@timvisee you still need to delete the generated snapshot files that you committed :)

**Comment by agourlay** [comment]
sorry for the fat finger :sweat_smile:

**Comment by timvisee** [comment]
> sorry for the fat finger 😅

My bad. Fixed!

**Review by agourlay — APPROVED** [review]

**Pull request merged**